src/spacecraftEmitter.ts

```diff
@@ -0,0 +1,22 @@
+import { EventEmitter } from 'events';
+import { event_messages } from './events_messages.json';
+
+export default class SpacecraftEmitter {
+	private eventEmitter: EventEmitter = new EventEmitter();
+	constructor(output: NodeJS.WriteStream) {
+		event_messages.forEach((message) => {
+			this.eventEmitter.on(message.id, () => {
+				output.write(` ${message.message} `);
+			});
+		});
```

Comment:
I believe it should be a method, to keep the constructor clean, with only a function call. What do you think?